src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
+  const sum = men.reduce((prev, man) => (man.died - man.born) + prev, 0);
+
+  return sum / men.length;
```

Comment:
create separate function to calculate the avarage age of array of the people and reuse it here and in `calculateWomenAverageAge`